### PR TITLE
9 get comments by review

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -241,4 +241,53 @@ describe("GET /api/users", () => {
...
+      .then(({ body: { comments } }) => {
+        expect(comments.length).toBeGreaterThanOrEqual(1);
+        comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("review_id");
+        });
```
Comment (Dutton1990) [inline __tests__/app.test.js:242]: Each comment should have the review_id, I would be explicit here and expect the review_id to be 2 along with these other expects.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -241,4 +241,53 @@ describe("GET /api/users", () => {
...
+      });
+  });
+  test("404: review_id does not exist", () => {
+    return request(app)
+      .get("/api/reviews/11111/comments")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Review 11111 has no comments");
```
Comment (Dutton1990) [inline __tests__/app.test.js]: I would change this message to be "Review 11111 does not exist"

**Comment by Dutton1990** [comment]
What if you're passed a review id that exists but has no comments attached yet? This should return a 200 and return an empty array. Write a test for it and see if you can implement it :)

**Review by douglashellowell — APPROVED** [review]
good work making those changes

feel free to merge!

**Pull request merged**